src/main/java/com/github/technus/tectech/Reference.java

```diff
@@ -12,6 +12,7 @@
     public static final String THAUMCRAFT = "Thaumcraft";
     public static final String DREAMCRAFT = "dreamcraft";
     public static final String GTPLUSPLUS = "miscutils";
+    public static final String SPARTAKCORE = "spartakcore";
```

Comment:
does it even require other recipes that default GT5u recipes from bloody loader?

Reply:
No, I  will write recipes in my core. Maybe you know I use a fork from GTNH GT and it's a reason for writing new core for deeper integration without bloody recipes. I hope you understand me.